dali/operators/generic/constant.cu

```diff
@@ -66,7 +66,8 @@ void FillTensorList(
 
     int n = tmp.size() * sizeof(Dst);
     int N = shape.num_samples();
-    cudaMemcpyAsync(dst.mutable_tensor<Dst>(0), tmp.data(), n, cudaMemcpyHostToDevice, stream);
+    CUDA_CALL(
```

Comment:
The part that calls a kernel isn't guarded.
Perhaps there should be a `CUDA_CALL(cudaGetLastError())` at call site instead, to capture all possible errors?